nym-vpn-lib/.cargo/config.toml

```diff
@@ -0,0 +1,8 @@
+[target.aarch64-linux-android]
+rustflags = ["-L", "../../main/jniLibs/arm64-v8a"]
+[target.x86_64-linux-android]
+rustflags = ["-L", "../../main/jniLibs/x86_64"]
+[target.armv7-linux-androideabi]
+rustflags = ["-L", "../../main/jniLibs/armeabi-v7a"]
+[target.i686-linux-android]
+rustflags = ["-L", "../../main/jniLibs/x86"]
```

Comment:
Aren't those referencing paths outside of this repo? I think we should avoid this, as it's dependent on each user's configuration of directories.